README.md

```diff
@@ -1,10 +1,58 @@
-# Sadulli
-vo SADULLI, in the kitchen garden grows parsley
+# Zubax Sadulli
 
-[Sadulli piccino](https://a360.co/2OxvigG)
+## Overview
 
-[Sadulli grosso](https://a360.co/2OwM1k0)
+Sadulli is a 500 W integrated drive for unmanned aerial applications that contains a motor and its control electronics in a compact enclosure. 
+
+The main design goal is to create a compact yet sophisticated motor controller that can be used with relatively small BLDC motors (about 40-45 mm in diameter) and can form an integrated drive together with the motor. This integrated drive should be possible to use as a direct replacement for just a motor alone without any major modifications to the aircraft it is used on. 
+
+## Brief specs
+
+- Operating voltage range 4-8S Li-Po (LiCoO<sub>2</sub>) battery (12-34 V)
+- Maximum continuous power 500 W
+- [UAVCAN hardware recommendations](https://uavcan.org/Specification/8._Hardware_design_recommendations/) following
+- Absence of necessity to invoke soldering to setup the device
+
+Sadulli consists of two main parts:  COTS motor and the ESC.  This repository contains the ESC hardware source files only.  There are three design variants of the Sadulli drive. All of them share the same ESC design.
+
+| Variant | Motor                                                        | Propeller diameter (inch) | Propeller pitch (inch) | Optimum thrust (kg) | Max thrust (kg) | Mass (g) |
+| ------- | ------------------------------------------------------------ | ------------------------- | ---------------------- | ------------------- | --------------- | -------- |
+| Grosso  | [V4014](http://en.rcsunnysky.com/v-multi-rotorefficiencytype/1083.html) | 17                        | 6.2                    | 1.5                 | 4               | 207      |
+| Piccino | [V4006](http://en.rcsunnysky.com/v-multi-rotorefficiencytype/1081.html) | 15                        | 5.5                    | 0.7                 | 1.5             | 128      |
+| Nudo    | N/A                                                          | N/A                       | N/A                    | N/A                 | N/A             | 62       |
+
+<p align="center">
+<img src="pics/nudo.png" alt="nudo" width = "40%" />
+<p align="center">
+<img src="pics/piccino.png" alt="piccino"  width = "40%" />
+<p align="center">
+<img src="pics/grosso.png" alt="grosso"  width = "40%" />
```

Comment:
```suggestion
<p align="center">
<img src="pics/nudo.png" alt="nudo" width="40%" />
<img src="pics/piccino.png" alt="piccino" width="40%" />
<img src="pics/grosso.png" alt="grosso" width="40%" />
</p>
```

Please make sure you close your tags properly.

Reply:
Also, the way the pics are arranged now is not great. Can we please put them in one row? You can use a table if you want, like I did here: https://github.com/Zubax/opencopcho#design

Reply:
*